contents/uix/uix.md

```diff
@@ -40,8 +42,72 @@ Here is a non-exhaustive list of UI design concepts you can choose to dive into
 
 ## UX Design
 
+At it's core, UX Design is about understanding both user and business goals, and tailoring a product that strikes a right balance between both goals within given constraints such as budget or time.
+
 Just like Software Development, UX Design is an iterative process. It involves many stages like researching, planning, testing, and at each stage are many different techniques that you could use. A good place to find these techniques would be the [UX Project Checklist](https://uxchecklist.github.io/) created by Andrea Soverini.
 
 ## Which is more important?
 
 The metaphor of building a house by Clayton Yan (UX Designer at UserTesting) illustrates this aptly. UX is the structure (i.e. the number of rooms, one-storey or two-storey and where the front door leads). Meanwhile, UI is about the visual (i.e. whether the same wallpaper or vases is used). A good UX with bad UI is like an ugly house with nice structure. A good UI with bad UX is like a beautiful house with the front door leading straight to the bathroom. To sum it up, UI and UX are **equally** important in creating an awesome experience for the users. 
+
+## The Process of UX Design
+
+As stated above, UX is a highly iterative process to achieve a fine balance between meeting the needs of the target audience and achieving business goals. Many times the functionality of a product is prioritised over the experience it gives. However this should not be the case when experience is the deciding factor in delighting or frustrating users.
+
+Catering the right experiences for users does not have to be difficult. The stages of UX Design are logical and can be decomposed into four main stages as shown, with each section supplemented with what you would do during each stage, and the expected deliverables at the end of each stage.
+1. Preliminary Research
+1. Prototyping
+1. User Testing
+1. Maintenance
+
+### Preliminary Research
+The first stage is Preliminary Research. This is where you would:
+  * Understand your target audience through [Contextual Inquiry](http://www.usabilitybok.org/contextual-inquiry), which involves interviewing users to find out about their wants, needs and pain-points, in the setting of them using the actual product and/or products from competitors
+  * Gather the tools needed for the project such as wireframing or prototyping [tools](https://medium.com/@Mockplus/the-10-best-wireframing-and-prototyping-tools-for-designers-a808e81ecadf) for creating low to high fidelity products for testing
+  * Depending on whether the project is hypothetical (taking a known product and re-designing it) or client-based (working for businesses or non-profits), discuss and create a project plan with stakeholders
+
+```Deliverables```
```

Comment:
Not sure code font is appropriate for this heading.

Reply:
Noted and changed